clients/src/main/java/org/apache/kafka/clients/producer/internals/SenderMetricsRegistry.java

```diff
@@ -84,7 +84,7 @@ public SenderMetricsRegistry(Metrics metrics) {
         this.batchSizeMax = createMetricName("batch-size-max",
                 "The max number of bytes sent per partition per-request.");
         this.compressionRateAvg = createMetricName("compression-rate-avg",
-                "The average compression rate of record batches.");
+                "The average compressed-to-uncompressed size ratio of record batches.");
```

Comment:
It might be a little more verbose, but perhaps we could phrase it like this?
```
The average compression rate of record batches, defined as the average ratio of the compressed batch size over the uncompressed size.
```

Reply:
I was in fact going for brevity with my approach, but if more detail will mean increased clarity for the user then I'm all for it. With the way you've defined it, I think there can be little question regarding the meaning.